src/Apps/Collect/Components/ArtworkGrid/CollectArtworkFilterRefetch.tsx

```diff
@@ -21,7 +21,7 @@ export class CollectArtworkFilterRefetch extends Component<
     Object.keys(this.props.filters).forEach(key => {
       if (
         key !== "page" &&
-        this.props.filters[key] !== prevProps.filters[key]
+        !isEqual(this.props.filters[key], prevProps.filters[key])
```

Comment:
when `key` is `time_periods` the value is an empty array but comparing two empty arrays for equality that don't have the same memory address returns `false`

```
// [] != []
```

Reply:
Makes sense! I wonder why this doesn't happen on the artist page? 🤔